bin/node/runtime/src/lib.rs

```diff
@@ -1942,6 +1942,14 @@ impl_runtime_apis! {
 		fn metadata() -> OpaqueMetadata {
 			OpaqueMetadata::new(Runtime::metadata().into())
 		}
+
+		fn metadata_at_version(version: u32) -> Option<OpaqueMetadata> {
+			Runtime::metadata_at_version(version)
+		}
+
+		fn metadata_versions() -> sp_std::vec::Vec<u32> {
+			Runtime::metadata_versions()
+		}
```

Comment:
When adding new methods you should really bump the version - the issue is that you now have a v1 with extra methods and a v1 without those methods. (So until this is exposed on metadata, it is problematic)